runtime/doc/eval.txt

```diff
@@ -4024,10 +4024,22 @@ jobsend({job}, {data})					{Nvim} *jobsend()*
 			:call jobsend(j, ["abc", "123\n456", ""])
 < 		will send "abc<NL>123<NUL>456<NL>".
 
-jobstart({name}, {prog}[, {argv}])			{Nvim} *jobstart()*
-		Spawns {prog} as a job and associate it with the {name} string,
-		which will be used to match the "filename pattern" in
-		|JobActivity| events. It returns:
+jobstart({argv}[, {opts}])			{Nvim} *jobstart()*
+		Spawns {argv}(list) as a job. If passed, {opts} must be a
+		dictionary with the any of the following keys:
+		  - on_stdout: stdout event handler
```

Comment:
I suggest you to use `on` prefix without underscores just like the HTML spec, also note that none of the vim functions have underscores although being dictionary keys here.